addon/-private/class/modifier-manager.ts

```diff
@@ -8,7 +8,7 @@ import ClassBasedModifier, {
   DESTROYED,
   InTeardown,
 } from './modifier';
-import { ModifierArgs } from 'ember-modifier/-private/interfaces';
+import { ModifierArgs } from '../interfaces';
```

Comment:
Imports like this would fail in the published package because standard node module resolution doesn't make a package visible to itself by name (testing as an installed package rather than using `paths` caught this)